ik:
  robot_description: '/robot_description'
  end_effector: 'iiwa_link_ee'
  name: 'iiwa_ik_server'